addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -154,9 +154,10 @@ export class PropertiesField extends Component {
         propertiesValues[targetIndex] = propertiesValues[propertyIndex];
         propertiesValues[propertyIndex] = prop;
         propertiesValues[propertyIndex].definition_changed = true;
-        this.props.update(propertiesValues);
-        // move the popover once the DOM is updated
-        this.shouldUpdatePopoverPosition = true;
+        this.props.update(propertiesValues).then(() => {
```

Comment:
Note: commit talks about [FIX] knowledge, but the fix is in web :)

Reply:
indeed, it was because the bug only occurred in knowledge for some reason, changed